ru/changelog.md

```diff
@@ -16,10 +16,26 @@
 
 ### yfm-transform
 
+#### 2.14.0
+
+- Добавлен санитайзинг сгенерированного HTML. По-умолчанию выключено. Параметр для включения: `needToSanitizeHtml: true`. Можно переопределить настройки по-умолчанию через параметр `sanitizeOptions`.
```

Comment:
Need add to https://ydocs.tech/ru/settings and add notice about different between `allowHTML` and `needToSanitizeHtml`